Pages/product.css

```diff
@@ -1,50 +1,68 @@
-.page-container{
+.page-container {
     display: flex;
 }
-.filters {
+
+.filter {
     width: 23%;
-    height: 100%;
+    height: 600px;
```

Comment:
Overall, the page looks very clean. Would suggest you to use rem instead of px for units.